doc/_templates/layout.html

```diff
@@ -0,0 +1,19 @@
+{% extends "!layout.html" %}
+
+{%- block extrahead %}
+
+   {% if theme_googletag  %}
```

Comment:
When we build from apidocs, we'll flipt to true. This way, people can clone this repo and it won't be on  whereas when we build for api.mongodb.com, it will be on and we can track.